Jenkinsfile

```diff
@@ -757,6 +766,30 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
                     }
                 }
             }
-        }
+	    // Save logs of failed builds to s3 - we want to analyze where we may have jenkins issues.
+	    script {
+                if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') {
+		    writeFile(file: 'build.log', text: getBuildLog())
+		    sh 'bin/clean-jenkins-log'
```

Comment:
Just realized that we should be passing the name of the log in as a parameter, instead of hard-coding `build.log` in the shell script. But whatever, we don't need to care about it.